prow/manifests/overlays/metal3/config.yaml

```diff
@@ -5,7 +5,7 @@ log_level: debug
 # Sinker configurations (for cleanup)
 sinker:
   resync_period: 1m
-  max_prowjob_age: 4h
+  max_prowjob_age: 24h
```

Comment:
24h seems way too long! Any particular reason to push it this long ?

Reply:
The default is [1 week](https://github.com/kubernetes-sigs/prow/blob/660eb6de4836e0027550877ab545cb6b625b771a/pkg/config/config.go#L1083-L1085). I don't think there is a reason to keep this low.
The prowjob is just some metadata in the cluster.

Reply:
Note also that `max_pod_age` below stays at 30m. That will clean up completed test pods

Reply:
Got it, thanks 
/lgtm